code/modules/reagents/newchem/drugs.dm

```diff
@@ -26,11 +26,45 @@
 	..()
 	return
 
-/datum/reagent/nicotine/overdose_process(var/mob/living/M as mob)
-	M.adjustToxLoss(1*REM)
-	M.adjustOxyLoss(1*REM)
-	..()
-	return
+/datum/reagent/nicotine/overdose_process(var/mob/living/M as mob, severity)
+	var/effect = ..()
+	if(severity == 1)
+		if(effect <= 2)
+			M.visible_message("<span class='warning'>[M] looks nervous!</span>")
+			M.confused += 15
+			M.adjustToxLoss(2)
+			M.Jitter(10)
+			M.emote("twitch_s")
+		else if(effect <= 4)
+			M.visible_message("<span class='warning'>[M] is all sweaty!</span>")
+			M.bodytemperature += rand(15,30)
+			M.adjustToxLoss(3)
+		else if(effect <= 7)
+			M.adjustToxLoss(4)
+			M.emote("twitch")
+			M.Jitter(10)
+	else if(severity == 2)
+		if(effect <= 2)
+			M.emote("gasp")
+			to_chat(M, "<span class='warning'>You can't breathe!</span>")
+			M.adjustOxyLoss(15)
+			M.adjustToxLoss(3)
+			M.Stun(1)
+		else if(effect <= 4)
+			to_chat(M, "<span class='warning'>You feel terrible!</span>")
+			M.emote("drool")
+			M.Jitter(10)
+			M.adjustToxLoss(5)
+			M.Weaken(1)
+			M.confused += 33
+		else if(effect <= 7)
+			M.emote("collapse")
+			to_chat(M, "<span class='warning'>Your heart is pounding!</span>")
+			M << sound('sound/goonstation/effects/heartbeat.ogg')
```

Comment:
why would you add a specific goon heart sound
we already have a heartbeat sound, use it